src/bci_build/package.py

```diff
@@ -155,9 +157,13 @@ def pretty_os_version_no_dash(self) -> str:
 
 #: Operating system versions that have the label ``com.suse.release-stage`` set
 #: to ``released``.
-RELEASED_OS_VERSIONS = [OsVersion.SP4, OsVersion.SP5, OsVersion.TUMBLEWEED]
+RELEASED_OS_VERSIONS = [OsVersion.SP5, OsVersion.TUMBLEWEED]
 
-ALL_OS_VERSIONS = [OsVersion.SP4, OsVersion.SP5, OsVersion.TUMBLEWEED]
```

Comment:
The bot itself is using this variable and is now broken due to this change.

Reply:
this has been fixed already.